Code/Framework/AzCore/AzCore/Math/Hemisphere.inl

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Math/Aabb.h>
+
+namespace AZ
+{
+    AZ_MATH_INLINE Hemisphere::Hemisphere(const Vector3& center, float radius, const Vector3& direction)
+        : m_centerRadius(AZ::Vector4(center, radius))
+        , m_direction(direction)
```

Comment:
Fair point. AZ::Plane just does a `AZ_MATH_ASSERT(normal.IsNormalized(), "This normal is not normalized");`, I think I'll follow their lead since I'd rather not take on work of normalizing when it might not be needed in a really low-level class like this.

Reply:
I'll also change the argument to `normalizedDirection` so people realize it needs to be normalized.